spec/controllers/spree/admin/search_controller_spec.rb

```diff
@@ -31,5 +31,29 @@
         end
       end
     end
+
```

Comment:
Need to add a spec that tests that a user cannot submit an unmanaged distributor_id and return customers of that enterprise.